docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -43,15 +38,14 @@ The IDE validates the connection and adds it to the Codewind panel:
 
 The IDE and Codewind are connected.
 
-# 2. Add a deployment registry
+## 2. Add a deployment registry.
```

Comment:
On line 21 `Add a deployment registry` was changed to `Add an image registry` so it should be changed here as well.